R/get_connection_PHS_database.R

```diff
@@ -17,21 +17,53 @@ phs_db_connection <- function(dsn, username = Sys.getenv("USER")) {
 
   # Check the username is not empty and take input if not
   if (is.na(username) | username == "") {
-    username <- rstudioapi::showPrompt(
-      title = "Username",
-      message = "Username",
-      default = ""
-    )
+    if (interactive()) {
```

Comment:
Using `rlang::is_interactive()` works the same in normal use, but will let us test things easier e.g. with 

```R
rlang::with_interactive({phs_db_connection("DVPROD")}, TRUE)
rlang::with_interactive({phs_db_connection("DVPROD")}, FALSE)
```

```suggestion
    if (rlang::is_interactive()) {
```